src/pages/en/docs/vision.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Vision
+description: Email Markup Consortium's vision for more consistent rendering in email clients.
+layout: "@layouts/MainLayout"
+---
+
+## Background
+
+Email clients have the challenge of dealing with third-party code: HTML email messages. For an email client to safely embed a HTML message, they need to:
+
+1. Sanitize the HTML message
+2. Embed the HTML message
+
+### Third-party content sanitization
+
+Third-party content sanitization is a problem any app/website must address if they wish to safely process or display third-party content. That is, only what is considered to be safe is kept and anything else is either removed or replaced.
+
+In the context of email, a HTML message is considered third-party content and email clients must sanitize this content to safely display it to the recipient. As things stand, each email client has their own way of sanitizing HTML messages. They mainly differ in two ways:
+
+1. Supported features: What features are considered safe to keep
+2. Unsupported features: How features that are not considered safe are handled
+
+Given each email client has their own implementation of (2), even if all email clients agreed on (1), there can still be major inconsistencies in how HTML messages are rendered in different email clients.
+
+#### Example
+
+At the time of writing, both Yahoo and Gmail do not support:
+
+1. CSS values using `rgb()` functional notation in the whitespace syntax
+2. `<picture>` element
+
+However, the rendered HTML in each email client is significantly different.
+
+Consider this snippet using the `rgb()` functional notation:
+
+```html
+<h1 style="text-transform:uppercase; font-size:1.3em; color:rgb(0 128 0 / .8);">
+  Heading
+</h1>
+```
+
+Yahoo removes the `color` property and its value because it does not support the `rgb()` functional notation in the whitespace syntax. It keeps everything as it.
```

Comment:
_"It keeps everything as it."_
Should that be
_"It keeps everything else as it is."_

Reply:
Good catch. Lunch time grammar mistakes!

Updated.